app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -81,6 +81,24 @@ def mfa_verify_totp(success:)
       )
     end
 
+    # Tracks when the user has attempted to verify via the WebAuthn-Platform MFA method to their account
+    # @param [Boolean] success
+    def mfa_verify_webauthn_platform(success:)
+      track_event(
+        :mfa_verify_webauthn_platform,
+        success: success,
+      )
+    end
+
+    # Tracks when the user has attempted to verify via the WebAuthn MFA method to their account
+    # @param [Boolean] success
+    def mfa_verify_webauthn(success:)
+      track_event(
+        :mfa_verify_webauthn,
```

Comment:
if we wanted to differentiate more clearly, we could call this `webauthn_roaming` which means more like a yubikey (something you can take from device to device, rather than it *being* the device), we use the term roaming in a few other spots: https://github.com/18F/identity-idp/search?q=roaming

Reply:
Nice - that is helpful!